integration_tests/data/expected/snowplow_unified_views_mobile_screen_engagement_expected.csv

```diff
@@ -0,0 +1,30 @@
+"view_id","engaged_time_in_s","absolute_time_in_s","horizontal_pixels_scrolled","vertical_pixels_scrolled","horizontal_percentage_scrolled","vertical_percentage_scrolled","last_list_item_index","list_items_count","list_items_percentage_scrolled"
+"FFA557C6-A1A4-41BB-ADD1-C3E22238CA8C",3.71,3.71,,,,,13,514,3
```

Comment:
Can you not quote the column names please, it causes some issues across warehouses sometimes